src/App/App.css

```diff
@@ -0,0 +1,70 @@
+.App {
+    text-align: center;
+}
+
+.App-logo {
+    height: 40vmin;
+    pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+    .App-logo {
+        animation: App-logo-spin infinite 20s linear;
+    }
+}
+
+h1,
+h2,
+h3 {
+    color: #f8f8f8;
+    font-weight: bolder;
+    background-color: #191716;
+    text-align: center;
+}
+
+h1 {
+    font-size: xxx-large;
+}
+
+h2 {
+    font-size: xx-large;
+}
+
+h3 {
```

Comment:
In here you are already setting the property `text-align: center` to the h3 element in line 22. So you can delete this selector